pkg/server/plugin/upstreamauthority/awssecret/awssecret.go

```diff
@@ -48,6 +53,35 @@ type Configuration struct {
 	AssumeRoleARN   string `hcl:"assume_role_arn" json:"assume_role_arn"`
 }
 
+func (p *Plugin) buildConfig(coreConfig catalog.CoreConfig, hclText string, status *pluginconf.Status) *Configuration {
+	newConfig := new(Configuration)
+	if err := hcl.Decode(newConfig, hclText); err != nil {
+		status.ReportError("plugin configuration is malformed")
+		return nil
+	}
+
+	env_session_token, env_session_token_exists := os.LookupEnv("AWS_SESSION_TOKEN")
```

Comment:
nit: does not follow naming conventions:
```suggestion
	envSessionToken, envSessionTokenExists := os.LookupEnv("AWS_SESSION_TOKEN")
```

Reply:
Fixed in next push, so I can sign it.